_posts/2020-06-01-Fortran-Newsletter-June-2020.md

```diff
@@ -0,0 +1,119 @@
+
+* [fortran-lang.org](#fortran-lang.org)
+* [Fortran Discourse](#fortran-discourse)
+* [Standard Library](#fortran-standard-library)
+* [Package Manager](#package-manager)
+* [Events](#events)
+
+## fortran-lang.org
+
+The Fortran website has been up since mid-April, and we've already got great
+feedback from the community already.
+In the past month we've updated the [Compilers](/compilers) page which is now
+comprehensive and includes all major open source and commercial compilers.
+The [Learn](/learn) page has also seen significant updates—it's been
+reorganized for easier navigation and currently features a quickstart tutorial,
+Fortran books, and other online resources.
+
+If you haven't yet, please explore the website and [let us know](https://github.com/fortran-lang/fortran-lang.org/issues)
+if you have any suggestions for improvement.
+Specifically, we'll be focusing on the [Learn](/learn) page and its mini-books
+in the coming months.
+Please help us make them better!
+
+Here are some specific items that we worked on:
+
+* [#90](https://github.com/fortran-lang/fortran-lang.org/pull/90)
+WIP: Mini-book on building programs
+* [#83](https://github.com/fortran-lang/fortran-lang.org/pull/83)
+Improving the structure and navigation of the [Learn](/learn) pages
+* [#46](https://github.com/fortran-lang/fortran-lang.org/pull/46)
+Build website previews from pull requests
+
+## Fortran Discourse
+
+On May 4 we launched the [Fortran Discourse](https://fortran-lang.discourse.group), an online discussion board
+for anything and everything Fortran related.
+You can use it discuss the Fortran language, ask for help, announce events and/or personal projects, or just lurk
+around. 
+There are already quite a few interesting discussions going on.
+Join us!
+
+## Fortran Standard Library
+
+Here's what's new in the Fortran Standard Library:
+
+* [#191](https://github.com/fortran-lang/stdlib/pull/191)
+WIP: Function for computing Pearson correlations among elements of
+an array in the `stdlib_experimental_stats` module
+* [#189](https://github.com/fortran-lang/stdlib/pull/189)
+WIP: Procedures for sparse matrices operations. Ongoing discussion on the API can be found
+[here](https://github.com/fortran-lang/stdlib/wiki/Stdlib-Sparse-matrix-API).
+* [#183](https://github.com/fortran-lang/stdlib/pull/183)
+Automatic API-doc generation and deployment of this [stdlib website](https://stdlib.fortran-lang.org)
+* [#170](https://github.com/fortran-lang/stdlib/pull/170)
+Addition of the new functions `diag`, `eye`, and `trace` functions to make working with
+matrices easier.
+Read the full specifications [here](https://stdlib.fortran-lang.org/page/specs/stdlib_experimental_linalg.html).
+
+## Package Manager
+
+In this past month support for dependencies between packages has been added
+to the [Fortran Package Manager (fpm)](https://github.com/fortran-lang/fpm).
+You can specify either a path to another folder on your machine with an fpm package,
```

Comment:
```suggestion
You can specify either a path to another folder on your machine with a fpm package,
```

Reply:
"an fpm" is okay here because its pronounced "ef-pi-em". See [this](https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html) article for example.

Reply:
Thanks for the explanation and link.